### PR TITLE
[DNM] Okd bootstrap only agent

### PR DESCRIPTION
**Pull request by andfasano**
Do not merge, just for test

### CODE REVIEW

**Comment by andfasano** [comment]
@LorbusChris okd aws jobs failed also in this test, relaunching them

**Comment by LorbusChris** [comment]
The SCOS test did bootstrap successfully though. I'll keep an eye on this

File: pkg/asset/ignition/bootstrap/common.go [hunk 1]
```diff
@@ -180,7 +180,7 @@ func (a *Common) generateConfig(dependencies asset.Parents, templateData *bootst
 	if err := AddSystemdUnits(a.Config, "bootstrap/systemd/common/units", templateData, commonEnabledServices); err != nil {
 		return err
 	}
-	if !templateData.IsOKD {
+	if !templateData.IsOKD || templateData.Invoker != "agent-installer" {
```
Comment (LorbusChris) [inline pkg/asset/ignition/bootstrap/common.go:183]: Should it just be this:?
```suggestion
	if templateData.Invoker != "agent-installer" {
```

Comment (LorbusChris) [reply pkg/asset/ignition/bootstrap/common.go:183]: no actually, nevermind.

**Pull request closed without merging**